src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,63 @@
+.calendar {
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: $day-font-size;
+
+  display: flex;
+  flex-wrap: wrap;
+  gap: $calendar-gap;
+  align-items: center;
+  align-content: center;
+
+  padding-inline: $calendar-padding;
+  width: $calendar-width;
+  height: 100vh;
+  margin: 0 auto;
+
+  &--start-day {
+    @each $day, $column in $weekdays {
+      &-#{$day} :first-child {
```

Comment:
There is an extra space before ':first-child'. It should be '&-#{$day}:first-child' without the space.